src/hubextensions.ts

```diff
@@ -17,6 +17,9 @@ type StartTransaction = (
 
 // Takes a transaction and determines if it should be profiled or not. If it should be profiled, it returns the
 // profile_id, otherwise returns undefined. Takes care of setting profile context on transaction as well
+/**
+ *
+ */
```

Comment:
```suggestion
```

Reply:
Will check all of these and add relevant comments, was just to satisfy the linters for now